src/types.ts

```diff
@@ -22,6 +22,12 @@ export interface IInputSignature {
   pubkey: Buffer;
 }
 
+export enum HistorySyncMode {
+  API = 'api',
```

Comment:
I'd use `POLLING_HTTP_API`, `MANUAL_STREAM_WS`, and `XPUB_STREAM_WS`.

Reply:
Done

Reply:
Thanks!